src/error.rs

```diff
@@ -68,6 +70,16 @@ impl Fail for Error {
     }
 }
 
+#[cfg(not(feature = "failure"))]

```

Comment:
Is there any downside to removing this condition and just always implementing `std::error::Error` ?

Reply:
`failure` automatically implements the `Fail` trait for all structs implementing `Error` which results in conflicts with the explicit `Fail` implementation.